repartitions/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from authentication.models import User
+
+
+class Campaign(models.Model):
+    STATUS = ("CLOSED", "OPEN", "RESULTS")
+
+    id = models.AutoField(primary_key=True)
+
+    name = models.CharField(max_length=200)
+    status = models.CharField(max_length=200, default="CLOSED")
+    even = models.BooleanField(
+        default=True
+    )  # should  all the categories be evenly split between groups
```

Comment:
This comment should go to `help_text`, at least it has a chance to be displayed somewhere in Django. Here, it will not even be displayed by the IDE's